packages/client/test/cli/cli-rpc.spec.ts

```diff
@@ -7,9 +7,8 @@ const cliArgs = ['--rpc', '--ws', '--dev', '--transports=rlpx']
 const end = (child: ChildProcessWithoutNullStreams, hasEnded: boolean, st: tape.Test) => {
   if (hasEnded) return
   hasEnded = true
-  child.stdout.removeAllListeners()
-  child.stderr.removeAllListeners()
```

Comment:
is it possible to keep these here in the end() function, but just after the new st.ok check?

Reply:
I just gave up on checking for the exit message.  If you remove them in function after the kill signal is sent, it removes them before the client prints the "Exiting..." message to the console so the test never registers.  But the `res` value of `true` should be just as good because that `SIGINT` is what we listen for in the client to shut it down and if the result we get back is something screwy, that return value should be `false` in our test.